src/ComScoreStreamingAnalytics.ts

```diff
@@ -39,7 +43,10 @@ export class ComScoreStreamingAnalytics {
 
     }
     this.registerPlayerEvents();
-
+    this.isMobile = !!navigator.userAgent && /Mobile/.test(navigator.userAgent);
```

Comment:
Please extract the `isMobile` check to some helper class. Can be static - no need to save this to the instance.

I'm not sure if this check is sufficient and cover all mobile devices. (It will not cover an iPad running iPadOS)